packages/core/analytics/core-analytics-browser-internal/src/analytics_service.ts

```diff
@@ -34,6 +35,8 @@ export class AnalyticsService {
     });
 
     this.registerBuildInfoAnalyticsContext(core);
+    // Register special `performance_metrics` type
```

Comment:
Nit: I'm not certain that the comment adds enough value here? Should we remove it?